packages/relay/src/lib/constants.ts

```diff
@@ -136,8 +136,8 @@ export default {
     DURATION: 60000,
   },
 
-  HBAR_RATE_LIMIT_DURATION: parseInt(process.env.HBAR_RATE_LIMIT_DURATION || '80000'),
-  HBAR_RATE_LIMIT_TINYBAR: parseInt(process.env.HBAR_RATE_LIMIT_TINYBAR || '11000000000'),
+  HBAR_RATE_LIMIT_DURATION: () => parseInt(process.env.HBAR_RATE_LIMIT_DURATION || '80000'),
+  HBAR_RATE_LIMIT_TINYBAR: () => parseInt(process.env.HBAR_RATE_LIMIT_TINYBAR || '11000000000'),
```

Comment:
Hmmm, most likely the .env file wasn’t imported in the tests before the `constants.ts` was imported. This change implies that env variables could change their value dynamically and we want to capture them only at startup of the relay and make them readonly from then on (please check Niki’s PR and also the issue which Eric has opened).

Revert this and just do a `dotenv.config({ path: <path_to_env_file> })` in the tests before you are import the `config.ts` file, the bug is coming from there. This won’t be needed after Niki’s PR is merged anyway.

Reply:
We can even make sure the env file is imported into the `process.env` always before it is accessed by `constants.ts` by calling `dotenv.config({ path: <path_to_env_file> })` directly inside `constants.ts` above any usages of `process.env`. This would fix the bug in all tests which don’t import it properly.

Reply:
Fair point! Reverted.